src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -63,7 +63,13 @@ class PanelMediaBrowser extends LitElement {
 
   @state() _currentItem?: MediaPlayerItem;
 
-  @state() _preferredLayout: MediaPlayerLayoutType = "auto";
+  @storage({
+    key: "mediaBrowserPreferredLayout",
+    storage: "localStorage",
+    state: true,
+    deserializer: (value) => value || "auto",
```

Comment:
We should not use `deserializer` for this, just a normal initialiser should work?

Reply:
https://github.com/home-assistant/frontend/pull/23107